server/config.go

```diff
@@ -10,21 +10,29 @@ import (
 	ini "gopkg.in/ini.v1"
 )
 
-// loadVarsFromINI will check the INI for the given environment and any environment variables
+// loadConfig will check the environment and the ini file for the config
 // It will give the following priority:
 // 1. Command-line value
 // 2. Environment Variable
 // 3. INI value
 // 4. Default value of flag
 // NOTE: All envvars need the prefix `CRIBBAGE_` and should exist in ALLCAPS.
 // NOTE: The INI file is pulled from the inis/ directory and is determined by the
-//       environment var `deploy` (set to `docker`, `prod`, etc.)
-func loadVarsFromINI() {
-	parseFlagsFromConfigFile(getConfigFile())
+// environment var `deploy` (set to `docker`, `prod`, etc.)
+func loadConfig() {
+	var confFileName string
+	if !isLambda() {
+		confFileName = getConfigFile()
+	}
+	parseConfig(confFileName)
 }
 
-func parseFlagsFromConfigFile(confFileName string) {
-	log.Printf("parseFlagsFromConfigFile from %q\n", confFileName)
+func parseConfig(confFileName string) {
+	if confFileName == `` {
+		log.Println(`parseConfig from environment only`, confFileName)
+	} else {
+		log.Printf("parseConfig from %q\n", confFileName)
```

Comment:
a man after my own heart: using backticks on all strings, except for the ones with backslashed characters in them. :heart_eyes:

Reply:
some say I learned from the best